### PR TITLE
Manual: fix documentation of the “fresh” tactic

### PR DESCRIPTION
**Pull request by vbgl**
-  The grammar is now accurate.
 -  The markup is now correct (w.r.t. the expected rendering).

### CODE REVIEW

File: doc/sphinx/proof-engine/ltac.rst [hunk 1]
```diff
@@ -107,7 +107,7 @@ mode but it can also be used in toplevel definitions as shown below.
                      : | solve [ `expr` | ... | `expr` ]
                      : | idtac [ `message_token` ... `message_token`]
                      : | fail [`natural`] [`message_token` ... `message_token`]
-                     : | fresh | fresh `string` | fresh `qualid`
+                     : | fresh [ `component` | … | `component` ]
```
Comment (Zimmi48) [inline doc/sphinx/proof-engine/ltac.rst]: I don't understand the use of the vertical bars. Shouldn't it be like for `idtac` above?

Comment (vbgl) [reply doc/sphinx/proof-engine/ltac.rst]: Indeed… I can’t remember where the meta-syntax is documented…

**Comment by Zimmi48** [comment]
>I can’t remember where the meta-syntax is documented…

Here is the (minimal) doc of the Sphinx `.. productionlist::` directive: http://www.sphinx-doc.org/en/stable/markup/para.html#directive-productionlist
and here is the Coq doc of the grammars in the refman: https://coq.inria.fr/refman/language/gallina-specification-language.html#about-the-grammars-in-the-manual

**Comment by vbgl** [comment]
Thanks for the pointers, Théo. I’ve thus addressed your comment.

I don’t understand the CI failure: seems spurious.

**Comment by Zimmi48** [comment]
Indeed, AppVeyor is a frequent source of spurious failures. It's just too close to the 1h timeout and in this case it went over the limit. Nothing to do with the doc which is not built in this job.

**Review by Zimmi48 — APPROVED** [review]

**Pull request merged**